### PR TITLE
doc: matter: rename network overview page

### PR DESCRIPTION
**Pull request by greg-fer**
Changed title of the network overview page.
Reworked a png image to svg.
KRKNWK-15224.

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/ug_matter_overview_network_topologies.rst [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
+####################################
 
 .. contents::
    :local:
    :depth: 2
 
-Matter network can be composed of Ethernet, Wi-Fi, and Thread devices.
-Networks are joined together, which means that devices that belong to different network can talk to each other within the local Matter fabric (infrastructure).
-All communication is achieved over IPv6, although the Matter network is capable of running in the absence of IPv6 infrastructure connected to the Internet.
+A Matter network can be composed of Ethernet, Wi-Fi, and Thread devices, which in turn can be part of one or more such IPv6-based networks.
```
Comment (kkasperczyk-no) [inline doc/nrf/ug_matter_overview_network_topologies.rst:10]: I don't quite understand the purpose of this sentence. Ethernet, Wi-Fi and Thread devices are always part of IPv6-based network, that usually are connected to another bigger IPv6 networks. I think that the previous version of this sentence was good enough.

File: doc/nrf/ug_matter_overview_network_topologies.rst [hunk 2]
```diff
@@ -1,32 +1,31 @@
...
+Matter joins these devices together within the local Matter fabric (infrastructure), which allows devices that belong to different network to talk to each other.
```
Comment (kkasperczyk-no) [inline doc/nrf/ug_matter_overview_network_topologies.rst:11]: That's not entirely true. All devices connected to the different networks (ethernet, Wi-Fi and Thread), but connected to one IP network are able to talk to each other over IP even without Matter. I would suggest to rephrase this sentence a little bit:
```suggestion
Matter joins these devices together within the local Matter fabric (infrastructure), which allows devices to talk to each other using the same Matter application layer, despite using different network technologies underneath.
```

**Review by kkasperczyk-no — COMMENTED** [review]
Why on the picture legend there are "Wi-Fi fabric" and "Thread fabric" names instead of "connection"? Fabric isn't a popular term regarding Wi-Fi and Thread.

File: doc/nrf/ug_matter_overview_network_topologies.rst [hunk 3]
```diff
@@ -1,32 +1,31 @@
...
 The Matter network topology refers to the structure of connections between Matter devices and IPv6 networks.
 Different IPv6 networks can communicate with each other through central hubs, for example Thread Border Router or Wi-Fi Access Point.
 
 Matter allows for connecting with external networks based on other protocols, such as Zigbee, through Matter bridges.
 
-The following figure illustrates a generic Matter topology example with two Thread Border Router devices, including one with an optional connection to cloud, and three Wi-Fi devices in the upper part of the figure acting as Matter bridges for other protocol devices.
+The following figure illustrates a generic Matter topology example with two Thread Border Router devices, including one with an optional connection to Internet, and three Wi-Fi devices in the upper part of the figure acting as Matter bridges for other protocol devices.
 
-.. figure:: images/matter_network_topologies.png
+.. figure:: images/matter_network_topologies.svg
```
[the remark this answers is not in the document]
Comment (greg-fer) [reply doc/nrf/ug_matter_overview_network_topologies.rst:28]: Fixed.

File: doc/nrf/ug_matter_overview_network_topologies.rst (hunk 1 above)
Comment (greg-fer) [reply doc/nrf/ug_matter_overview_network_topologies.rst:10]: Brought back the first version of the sentence plus minor additions.

**Comment by greg-fer** [comment]
@kkasperczyk-no , @Damian-Nordic , please recheck.

**Review by Damian-Nordic — APPROVED** [review]

**Review by kkasperczyk-no — APPROVED** [review]

**Pull request merged**